src/client/algod.js

```diff
@@ -21,17 +21,25 @@ function Algod(token, baseServer = "http://r2.algorand.network", port = 4180) {
      * @returns {Promise<*>}
      */
     this.status = async function () {
-        let res = await c.get("/v1/status");
-        return res.body;
+        try {
```

Comment:
Seems like it worth trying to put the try catch mechanism in a separate function.

Reply:
@rotemh I initially agreed but after trying this, I think it might be cleaner/shorter as-is. For context, I was trying to implement some sort of 
```
tryAndCatchWrapper = function(func) {
    try {
        func()
    } catch (e) {
      //context-dependent re-throw logic
    }
}
```
And then each `algod.js` function would pass its logic as `func`, but that quickly ended up more verbose and confusing than just using `try`/`catch`.

Reply:
I thought of wrapping the entire get functionality so you'd pass just the path, but I accept that. It's not a big deal.